daffodil-lib/src/test/scala/org/apache/daffodil/lib/util/TestListSerialization.scala

```diff
@@ -22,7 +22,7 @@ import org.junit.Assert.fail
 import org.junit.Test
 
 class Thing() extends Serializable {
-  var things: Seq[Thing] = _
+  var things: collection.Seq[Thing] = _
```

Comment:
More collection.Seq instead of just Seq; is this necessary?  I don't think it is.

Reply:
I think you're right. In this case, I can leave it as Seq, but I need to change the assert to not look for Collection.Seq, because in 2.13, the value will be Immutable.Seq.